.github/workflows/molecule.yml

```diff
@@ -0,0 +1,44 @@
+---
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          path: "${{ github.repository }}"
+      - name: molecule
+        uses: robertdebock/molecule-action@2.6.3
+        with:
+          command: lint
+  test:
+    needs:
+      - lint
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
```

Comment:
This role [also supports amazon linux](https://github.com/AcroMedia/ansible-role-new-relic/blob/master/meta/main.yml#L8) (v2018, and v2). Geerlinguy has a [docker built for v2](https://github.com/geerlingguy/docker-amazonlinux2-ansible). We can skip the 2018 version, since it's based on RHEL6, which is EOL this month.